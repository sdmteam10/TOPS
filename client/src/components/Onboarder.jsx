import React, { Component } from 'react';

class Onboarder extends Component {
  render() {
    return (
      <div className="mt-md-5 ws-0 mx-md-auto">
        <div>
          <h1 className="text-secondary fw-light font-monospace">Welcome Onboarder!</h1>
          <br />
          <br />
        </div>
        <div className="font-monospace">Onboarder Page</div>
      </div>
    )
  }
}

export default Onboarder