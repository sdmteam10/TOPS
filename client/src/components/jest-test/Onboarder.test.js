// Onboarder.test.js
import React from 'react'
import Onboarder from '../Onboarder'
import renderer from 'react-test-renderer';
import { fireEvent, render } from "@testing-library/react"

//it.only() "only test this <it>."
//it.skip() "not test this <it>."
// done() under expect(){} -  should be only used for async functions

  describe('Test <Onboarder /> component', function() {
    it('Should render', function() {  
      //Render component should be successed.
      const component = renderer.create(
        <Onboarder />
      );
        let testSnap = component.toJSON();
        expect(testSnap).toMatchSnapshot(); 
    })
  })