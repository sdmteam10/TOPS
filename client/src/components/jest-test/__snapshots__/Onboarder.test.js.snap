// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Onboarder /> component Should render 1`] = `
<div>
  <div
    className="mt-md-5 ws-0 mx-md-auto"
  >
    <div>
      <h1
        className="text-secondary fw-light font-monospace"
      >
        Welcome Onboarder!
      </h1>
      <br />
      <br />
    </div>
    <div>
      <a
        className="link-secondary fw-light font-monospace"
        href="OnboarderInfo"
      >
        Onboarder Information Form
      </a>
    </div>
  </div>
</div>
`;
